### PR TITLE
add note regarding new module

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/523

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR. Thanks.

**Comment by dariomws306** [comment]
I think the note is not bad... But replace New-MsolServicePrincipalCredential with New-AzureADServicePrincipalKeyCredential in the example could be a better improvement for this article.

Let's see what @get-itips and @tseward think about it.

File: docset/windows/adfs/new-adfsazuremfatenantcertificate.md [hunk 1]
```diff
@@ -52,6 +52,9 @@ PS C:\> Set-AdfsAzureMfaTenant -TenantId <your tenant ID> -ClientId 981f26a1-7f4
 
 These commands create a certificate for Azure MFA, register the certificate in a tenant, and enable Azure MFA on an AD FS farm.
 
+> [!NOTE]
+> Customers are encouraged to use the newer Azure Active Directory V2 PowerShell module. For more information about the V2 module please see [Azure Active Directory V2 PowerShell](https://docs.microsoft.com/en-us/powershell/module/Azuread/?view=azureadps-2.0).
```
Comment (nenonix) [inline docset/windows/adfs/new-adfsazuremfatenantcertificate.md]:
```suggestion
> Customers are encouraged to use the newer Azure Active Directory PowerShell 2.0 module. For more information about the v2.0 module please see [AzureAD PowerShell 2.0](https://docs.microsoft.com/powershell/module/Azuread/?view=azureadps-2.0).
```

**Comment by tseward** [comment]
@dariomws306 Agreed. Otherwise OK.

**Comment by get-itips** [comment]
Agreed with @dariomws306

**Comment by VLG17** [comment]
> I think the note is not bad... But replace New-MsolServicePrincipalCredential with New-AzureADServicePrincipalKeyCredential in the example could be a better improvement for this article.
> 
> Let's see what @get-itips and @tseward think about it.

@dariomws306 @get-itips @tseward I thought of that before adding the note, but i could not find an equivalent for all the parameter values of the old cmdlet (New-MsolServicePrincipalCredential).

I can't tell from the public docs if the "Type" and "Usage" parameters for "New-AzureADServicePrincipalKeyCredential" still accept the "old" values: "asymmetric", "symmetric", "password"

I did not want to assume it's the same, so i went with the note.

**Comment by get-itips** [comment]
Hello @VLG17 

Type accepts: AsymmetricX509Cert,  Symmetric
Usage accepts: Sign, Verify

**Comment by e0i** [comment]
@VLG17 Please follow-up with Andres's suggestions to move forward. Thanks.

**Comment by tseward** [comment]
Looks good.

**Comment by get-itips** [comment]
Looks ok

**Comment by e0i** [comment]
@AndreaBarr The PR that has passed both copy end technical review is ready to be merged. Thanks.

**Pull request merged**